src/app/app.component.ts

```diff
@@ -276,12 +276,13 @@ export class AppComponent {
       return;
     }
     const patientBundle = createPatientBundle(this.searchReqObject, this.bundleResources);
-    this.searchService.searchClinicalTrials(patientBundle).subscribe(
+    this.searchService.searchAllTrials(patientBundle).subscribe(
       (response) => {
         // Store the results
-
-        this.searchResults = response;
-        console.log(response);
+        //let mergedResponse =response;
+        let mergedResponse = this.searchService.mergeSearchBundles(response);
+        this.searchResults = mergedResponse;
+        console.log(mergedResponse);
```

Comment:
This is nitpicking, but we should remove the commented line and technically `mergedResponse` should be `const` (as the linter will say). But ultimately if it's only used to locally `console.log` the results you could just store it straight into `this.searchResults` and `console.log` that.